### ISSUE TITLE
WCAG 2.5.5: Touch inputs must have a sufficient target size. (android.view.View)

### ISSUE THREAD

**Issue by petra1** [petra1/AccessibilityDemo#12]
#### Issue
`Touch inputs must have a sufficient target size.` ([TouchSizeWcag](https://accessibilityinsights.io/info-examples/android/touch-size-wcag/))
#### Target application
ch.petraaccess_for_all.accessibilitydemo
#### Element path
android.view.View
#### How to fix
    The element has an insufficient target size (width: 201.142dp, height: 23.238dp). Set the element's minWidth and minHeight attributes to at least 44dp.
![grafik](https://user-images.githubusercontent.com/10880024/119220312-3937bd80-baea-11eb-9e07-169faf8812ac.png)
